modules/kafka/src/main/java/org/testcontainers/containers/KafkaContainer.java

```diff
@@ -139,8 +139,8 @@ protected void containerIsStarting(InspectContainerResponse containerInfo, boole
     private String startZookeeper() {
         ExecCreateCmdResponse execCreateCmdResponse = dockerClient.execCreateCmd(getContainerId())
             .withCmd("sh", "-c", "" +
-                "printf 'clientPort=" + ZOOKEEPER_PORT + "\ndataDir=/var/lib/zookeeper/data\ndataLogDir=/var/lib/zookeeper/log' > /zookeeper.properties\n" +
-                "zookeeper-server-start /zookeeper.properties\n"
+                "printf 'clientPort=" + ZOOKEEPER_PORT + "\ndataDir=/var/lib/zookeeper/data\ndataLogDir=/var/lib/zookeeper/log' > zookeeper.properties\n" +
```

Comment:
These `\n`s within the string seem likely to lead to mistakes. As we're editing it, should we break this apart?

e.g.
```suggestion
                "printf 'clientPort=" + ZOOKEEPER_PORT + "\n" +
                "dataDir=/var/lib/zookeeper/data\n" +
                "dataLogDir=/var/lib/zookeeper/log' > zookeeper.properties\n" +
```
(🙏 save us, text blocks...)